FogCreek/Glitch-Community
# Electric check: clear out old popovers

## Links
* Remix link: https://electric-check.glitch.me/

## Changes:
* remove unused "legacy" popovers & styles
* inline the one use of dropdown into create-collection-pop

## How To Test:
* log into https://electric-check.glitch.me/
* select "Add to Collection" from a project popup
* click "Add to a new collection" button
* verify the collection owner dropdown renders correctly
